src/style.css

```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: #CCFF0C;
```

Comment:
Since we are applying style to `body` in `MainCss.elm`, can't we move this style to it and delete the `src/style.css`?